### PR TITLE
Fix nginx upstreams

### PR DESCRIPTION
**Pull request by aperigault**
##### SUMMARY
Fix issue #1452 

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - Installer

##### AWX VERSION
```
1.0.4.0
```

### CODE REVIEW

File: awx/sso/pipeline.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def prevent_inactive_login(backend, details, user=None, *args, **kwargs):
         raise AuthInactive(backend)
 
 
-def _update_m2m_from_expression(user, rel, expr, remove=True):
+def _update_m2m_from_expression(user, rel, expr, remove=True, saml_team_names=False):
```
Comment (ryanpetrello) [inline awx/sso/pipeline.py]: @aperigault did you intend to commit this?  It looks unrelated to the PR subject.

Comment (aperigault) [reply awx/sso/pipeline.py]: Sorry its related to another PR.
Fixed.

**Review by wwitzel3 — APPROVED** [review]
